components/coding/studentPortal/Sidebar.tsx

```diff
@@ -16,97 +16,95 @@ export const Sidebar: React.FC<SidebarProps> = ({}: SidebarProps) => {
             >
               <div className="flex flex-wrap">
                 <img
                   className="w-8 h-8 mr-3"
                   src={
-                    isStageActive("Classroom")
-                      ? "/images/classroomActive.svg"
-                      : "/images/classroomInactive.svg"
+                    isStageActive("Dashboard")
+                      ? "/images/dashBoardActive.svg"
+                      : "/images/dashBoardInactive.svg"
                   }
                 />
-                Classroom
+                Dashboard
               </div>
             </div>
-
-            <div
-              className={`p-4 border-l-4 ${
-                isStageActive("Profile")
-                  ? "border-charmander text-charmander"
-                  : ""
-              }`}
-              onClick={() => onStageClick("Profile")}
-            >
-              <div className="flex flex-wrap">
-                <img
-                  className="w-7 h-7 mr-3"
-                  src={
-                    isStageActive("Profile")
-                      ? "/images/profileActive.svg"
-                      : "/images/profileInactive.svg"
-                  }
-                />
-                Profile
-              </div>
+          </div>
+          <div
+            className={`p-4 border-l-4 ${
+              isStageActive("Classroom")
+                ? "border-charmander text-charmander"
+                : ""
+            }`}
+            onClick={() => onStageClick("Classroom")}
+          >
+            <div className="flex flex-wrap">
+              <img
+                className="w-8 h-8 mr-3"
+                src={
+                  isStageActive("Classroom")
+                    ? "/images/classroomActive.svg"
+                    : "/images/classroomInactive.svg"
+                }
+              />
+              Classroom
             </div>
           </div>
-          <div className="flex flex-col justify-end mb-18">
-            <div className="flex flex-wrap ">
-              <img className="w-7 h-7 mr-3" src="/images/logoutButton.svg" />
-              Logout
+
+          <div
+            className={`p-4 border-l-4 ${
+              isStageActive("Profile")
+                ? "border-charmander text-charmander"
+                : ""
+            }`}
+            onClick={() => onStageClick("Profile")}
+          >
+            <div className="flex flex-wrap">
+              <img
+                className="w-7 h-7 mr-3"
+                src={
+                  isStageActive("Profile")
+                    ? "/images/profileActive.svg"
+                    : "/images/profileInactive.svg"
+                }
+              />
+              Profile
             </div>
           </div>
         </div>
+        <div className="flex flex-col justify-end mb-18">
+          <div className="flex flex-wrap ">
+            <img className="w-7 h-7 mr-3" src="/images/logoutButton.svg" />
```

Comment:
stick to multiples of 4 as best you can for tailwind units. ie use w-8 instead of w-7, and mr-4 instead of mr-3